baitap-19-10-2020/bai-1.js

```diff
@@ -0,0 +1,17 @@
+const salaryCalculation = (salary) => {
+    if(salary >= 15000000)
+    {

```

Comment:
```suggestion
    if (salary >= 15000000) {
```
1. Nhìn kỹ cách đặt dấu cách
2. indent là 2 spãce